core/server/api/v2/utils/validators/input/schemas/tags.json

```diff
@@ -0,0 +1,63 @@
+
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "tags",
+    "title": "tags",
+    "description": "Base tags definitions",
+    "definitions": {
+        "tag": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "maxLength": 191,
+                    "pattern": "^([^,]|$)"
+                },
+                "slug": {
+                    "type": ["string", "null"],
+                    "maxLength": 191
+                },
+                "description": {
+                    "type": ["string", "null"],
+                    "maxLength": 500
+                },
+                "feature_image": {
+                    "type": ["string", "null"],
+                    "format": "uri-reference",
+                    "maxLength": 2000
+                },
+                "visibility": {
+                    "type": "string",
+                    "enum": ["public", "internal"]
+                },
+                "meta_title": {
+                    "type": ["string", "null"],
+                    "maxLength": 300
+                },
+                "meta_description": {
+                    "type": ["string", "null"],
+                    "maxLength": 500
+                },
+                "id": {
```

Comment:
Not allowing for custom `id` anymore, previously any random string could be assigned. This is a similar behavior which was fixed in `/posts`

Reply:
Cool i guess we close https://github.com/TryGhost/Ghost/issues/9100 then

Reply:
Yeap, prepared a comment there once this lands in master already :+1: